src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -22,118 +24,93 @@ import * as CardUtils from '@libs/CardUtils';
 
     /** React ref being forwarded to the PaymentMethodList Button */
-    buttonRef: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
+    buttonRef?: Ref<View>;
 
     /** To enable/disable scrolling */
-    shouldEnableScroll: PropTypes.bool,
+    shouldEnableScroll?: boolean;
 
     /** List container style */
-    style: stylePropTypes,
+    style?: StyleProp<ViewStyle>;
+
+    /** Type to filter the payment Method list */
+    filterType?: TupleToUnion<typeof FILTER_TYPES>;
+    /** Whether the add bank account button should be shown on the list */
+    shouldShowAddBankAccount?: boolean;
+
+    /** Whether the add Payment button be shown on the list */
+    shouldShowAddPaymentMethodButton?: boolean;
+
+    /** Whether the assigned cards should be shown on the list */
+    shouldShowAssignedCards?: boolean;
+
+    /** Whether the empty list message should be shown when the list is empty */
+    shouldShowEmptyListMessage?: boolean;
+
+    /** What to do when a menu item is pressed */
+    onPress: (event?: GestureResponderEvent | KeyboardEvent, accountType?: string, accountData?: AccountData, icon?: IconAsset, isDefault?: boolean, methodID?: number) => void;
 };
 
-const defaultProps = {
-    bankAccountList: {},
-    cardList: {},
-    fundList: null,
-    userWallet: {
-        walletLinkedAccountID: 0,
-        walletLinkedAccountType: '',
-    },
-    isLoadingPaymentMethods: true,
-    shouldShowAddBankAccount: true,
-    shouldShowAddPaymentMethodButton: true,
-    shouldShowAssignedCards: false,
-    shouldShowEmptyListMessage: true,
-    filterType: '',
-    actionPaymentMethodType: '',
-    activePaymentMethodID: '',
-    selectedMethodID: '',
-    listHeaderComponent: null,
-    buttonRef: () => {},
-    onListContentSizeChange: () => {},
-    shouldEnableScroll: true,
-    style: {},
-    shouldShowSelectedState: false,
+type PaymentMethodItem = PaymentMethod & {
+    onPress?: (e: GestureResponderEvent | KeyboardEvent | undefined) => void;
+    canDismissError?: boolean;
+    disabled?: boolean;
+    shouldShowRightIcon?: boolean;
+    interactive?: boolean;
+    brickRoadIndicator?: ValueOf<typeof CONST.BRICK_ROAD_INDICATOR_STATUS>;
 };
 
-/**
- * Dismisses the error on the payment method
- * @param {Object} item
- */
-function dismissError(item) {
+function dismissError(item: PaymentMethod) {
     const isBankAccount = item.accountType === CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT;
     const paymentList = isBankAccount ? ONYXKEYS.BANK_ACCOUNT_LIST : ONYXKEYS.FUND_LIST;
-    const paymentID = isBankAccount ? lodashGet(item, ['accountData', 'bankAccountID'], '') : lodashGet(item, ['accountData', 'fundID'], '');
+    const paymentID = isBankAccount ? item.accountData?.bankAccountID ?? '' : item.accountData?.fundID ?? '';
 
     if (!paymentID) {
-        Log.info('Unable to clear payment method error: ', item);
+        Log.info('Unable to clear payment method error: ', undefined, item);
```

Comment:
why?

Reply:
So we're fixing an unrelated small bug, right?